lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -97,22 +119,34 @@ def _init_job_rec(
 
         inputs.app_id = params.get(_APP_ID)
         inputs.source_ws_objects = params.get(_SOURCE_WS_OBJECTS)
-        inputs.parent_job_id = str(params.get(_PARENT_JOB_ID))
+
+        parent_job_id = params.get(_PARENT_JOB_ID)
```

Comment:
Bugfix, we shouldn't always add a "None" parent_id

Reply:
I've been wondering about that for a while